config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-ci.yaml

```diff
@@ -46,6 +28,10 @@ periodics:
         value: "us-west-2"
       securityContext:
         privileged: true
+      resources:
+        requests:
+          cpu: 1
+          memory: "4Gi"
```

Comment:
Attempting to add requests to see if it helps to clear up some of the flakiness we've seen around e2e tests previously.